config/locales/en/shared.yml

```diff
@@ -429,6 +429,20 @@ en:
       housing_benefit:
         heading: Housing Benefit
         does_your_client: Does %{individual} get Housing Benefit?
+      linking_and_copying:
```

Comment:
If this is now loaded only as the partial, can the original locale entries be removed?

Reply:
YESSSS

I knew I'd forgotten something! Good spot